pds_doi_service/core/actions/draft.py

```diff
@@ -333,8 +333,13 @@ def _run_single_file(self, input_file, node, submitter, force_flag, keywords):
              exception_messages) = collect_exception_classes_and_messages(
                 err, exception_classes, exception_messages
             )
-        # Catch all other exceptions as errors
+        # Propagate input format exceptions, force flag should not affect
+        # these being raised and certain callers (such as the API) look
+        # for this exception specifically
         except InputFormatException as err:
```

Comment:
Is the input format exception one of the criticalDOIException ? I would have it among them.